docs/docs/en/guide/start/quick-start.md

```diff
@@ -1,61 +1,118 @@
 # Quick Start
 
-This is a Quick Start guide to help you get a basic idea of working with Apache DolphinScheduler. Once you've gone through the tutorial video, we encourage you to find out more about Apache DolphinScheduler functions and examples in .
+In this section, we will use DolphinScheduler to create and run a simple workflow step by step. During this journey,
+you will learn the basic concepts of DolphinScheduler and know the most basic configuration to run the workflow. We
+provide both video and text in this tutorial, you can choose the way you prefer
+
+## Video Tutorial
 
 <figure class="video_container"> 
   <iframe src="https://www.youtube.com/embed/nrF20hpCkug" frameborder="0" allowfullscreen="true"></iframe>
 </figure>
 
-## Administrator User Login
+## Text Tutorial
+
+### Setup Dolphinscheduler
+
+You have to install and start dolphinscheduler first before go ahead. For the beginner, we recommend setup up
+dolphionscheduler with the official Docker image or starting the standalone server.
+
+* [standalone server](https://dolphinscheduler.apache.org/en-us/docs/3.1.3/guide/installation/standalone)
+* [docker](https://dolphinscheduler.apache.org/en-us/docs/3.1.3/guide/start/docker)
+
+### Build Your First Workflow
+
+You can log in dolphinscheduler with http://localhost:12345/dolphinscheduler/ui the default username/password
+is `admin/dolphinscheduler123`.
+
+![login](../../../../img/start/login.gif)
+
+#### Create Tenant
+
+Tenant is a concept that cannot be bypassed while using DolphinScheduler, so
+let's briefly introduce the concept of tenant first.
+
+The account named `admin` logged into DolphinScheduler is called user in dolphinscheduler.
```

Comment:
DolphinScheduler maps the `admin` account you use to log into DolphinScheduler to `user`.

Reply:
Add in https://github.com/apache/dolphinscheduler/pull/13607#discussion_r1119528273